### PR TITLE
[MOMNG-843] Start of pathfinder report

### PR DESCRIPTION
**Pull request by alexmuller**
This is the scaffold and start of the functionality to add a new kind of report for a Pathfinder cohort study.

Commit detail:

##  Remove registration-fixes.txt

I think this file is unused

## Cache container image after installing dependencies

This makes the build faster in development because we don't have to keep running `pip install`.

##  Add endpoint to list kinds of reports 

Could be used in the control-interface.

## Add start of new task for generating a message report 

Needs more work to fetch the details from the message-sender.

##  Add view method for creating new report 

`POST` to create a task which generates this new report.

### CODE REVIEW

File: reports/views.py [hunk 1]
```diff
@@ -56,6 +61,40 @@ def post(self, request, *args, **kwargs):
...
+
+    POST: Start generating a report.
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        request_data = request.data.copy()
+
+        request_data['start_date'] = '2017-09-01'
+        request_data['end_date'] = '2018-09-01'
```
Comment (KaitCrawford) [inline reports/views.py:76]: I was thinking we could make this slightly more generic and the user could specify the dates. Would that complicate things?

**Review by KaitCrawford — COMMENTED** [review]
looking good! Just one comment

**Comment by KaitCrawford** [comment]
@alexmuller I think I'm going to land this PR and build on top of it. If I just continue with this on the same branch then this PR is going to get very bloated and I'd rather have the scaffold you've added in place

**Comment by alexmuller** [comment]
@KaitCrawford I think that's probably easier, thanks!

**Pull request merged**